pkg/provision/primitives/container.go

```diff
@@ -249,6 +253,13 @@ func (p *Provisioner) containerProvisionImpl(ctx context.Context, reservation *p
 		return ContainerResult{}, err
 	}
 
+	var elevated = false
+
+	if config.FList == FListElevated {
```

Comment:
I think it's a quite useful feature and can see it used more in the future for other things, I guess It might be a good idea to have the elevated information on the flist object itself on the hub, and based on its permissions it'd reflect here as elevated.

Reply:
We don't want anybody to get theses elevated privileged, only trusted flist.